app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -3,28 +3,45 @@
 @import '../../../sizes'
 
 table
+  display: grid
+  grid-template-columns: 1fr min-content min-content min-content min-content
   border: solid 1px $maincolor
   border-collapse: collapse
+
+
+
   thead
+    display: contents
     border: solid 1px $maincolor
-  td, th
+  tbody,tr
+    display: contents
+  tr:hover td
+    background-color: $softbgcolor
+  th
+    border-bottom: solid 1px $maincolor
+    background-color: $softbgcolor
     padding: 2px 4px
   th.manage
     padding: 2px 0
-  tr.tableheader
-    background-color: $softbgcolor
   tr.poll
     td
+      padding: 2px 4px
```

Comment:
Before your modifications, it was clear that th and td must have the same padding. This is not the case any more. Please fix that.